static/icons/nuxt-google-sheets-parser.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generator: Adobe Illustrator 25.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 64 88" style="enable-background:new 0 0 64 88;" xml:space="preserve">
```

Comment:
I recommend you to have a svg directly in 40*40px because it will be reduce at this size otherwise

Reply:
I really thanks for your help here, too!
I also fix this line but honestly I don't know about svg attributes properly.
But I could see this logo's container sizes 40x40 on browser.
If I was wrong, I'll fix it.
Thanks Jimmy!